packages/components/atoms/Button/Button.stories.tsx

```diff
@@ -24,23 +24,23 @@ export default {
       },
     },
     type: {
-      description: 'Primary colors',
+      description: 'Button type',
       control: { type: 'text' },
```

Comment:
```suggestion
      control: 'select',
```

This should work for showing a dropdown instead of a text input in storybook. The types need to be well set though for this to work correctly.

Reply:
Great! this is much better